extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -506,19 +506,16 @@ class CornerstoneViewportService extends PubSubService
       !hangingProtocolService.customImageLoadPerformed
     ) {
       // delegate the volume loading to the hanging protocol service if it has a custom image load strategy
-      if (
-        hangingProtocolService.runImageLoadStrategy({
-          viewportId: viewport.id,
-          volumeInputArray,
-        })
-      ) {
-        // Fallback to the default strategy if the custom one fails
-        return;
-      }
+      return hangingProtocolService.runImageLoadStrategy({
```

Comment:
we should not fallback to default, the imageLoadStrategy should handle when it is ready to continue, not here, if customImageLoad is defined we should not .load anymore